client/src/pages/CreateWorkout/CreateWorkout.jsx

```diff
@@ -127,7 +137,7 @@ const CreateWorkout = () => {
 
         <div className="flex w-full grow flex-col items-center gap-3">
           {" "}
-          <ListExercises />
+          <ListExercises workout={workout} setWorkout={setWorkout} />
```

Comment:
Hmm instead of passing the entire workout & setWorkout here, ListExercises rly only cares about exercises so could take just that field + something specific like `handleExercisesChange`